task3/task3/src/Main.java

```diff
@@ -0,0 +1,85 @@
+        Scanner sc = new Scanner(System.in);
+        String comm = "exit";
+        String command = "";
+        while (!command.equals(comm)) {
+
+            String operator;
+            double result;
+            System.out.println("Please input first number");
+            double first = CorrectNumb();
+
+            System.out.println("Please input operation");
+            operator = sc.next();
+            String plus = "+";
+            String mines = "-";
+            String divide = "/";
+            String multiply = "*";
+            String percent = "%";
+
+            if (operator.equals(plus)) {
+                System.out.println("Please input second number");
+                double second = CorrectNumb();
+                result = first + second;
+                System.out.println(first + operator + second + "=" + result);
+            } else {
+                if (operator.equals(mines)) {
+                    System.out.println("Please input second number");
+                    double second = CorrectNumb();
+                    result = first - second;
+                    System.out.println(first + operator + second + "=" + result);
+                } else {
+                    if (operator.equals(multiply)) {
+                        System.out.println("Please input second number");
+                        double second = CorrectNumb();
+                        result = first * second;
+                        System.out.println(first + operator + second + "=" + result);
+                    } else {
+                        if (operator.equals(divide)) {
+                            System.out.println("Please input second number");
+                            double second = CorrectNumb();
+                            if (second != 0) {
+                                result = first / second;
+                                System.out.println(first + operator + second + "=" + result);
+                            } else {
+                                System.out.println("Error: divide by 0 is not allowed!");
+                            }
+                        } else {
+                            if (operator.equals(percent)) {
+                                System.out.println("Please input second number");
+                                double second = CorrectNumb();
+                                result = first % second;
+                                System.out.println(first + operator + second + "=" + result);
+                            } else {
+                                System.out.println("Please input correct operator");
+                            }
+                        }
+                    }
+                }
+            }
+            System.out.println("Enter " + " exit:" + " to close!");
+            sc.nextLine();
+            command = sc.nextLine();
+            command = command.toLowerCase();
+        }
+    }
+    public static double CorrectNumb() {
```

Comment:
first latter of method name should be lower case

also method should describe functionality (for example getDoubleInput())